nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -128,6 +128,8 @@
 
 - [wg-access-server](https://github.com/freifunkMUC/wg-access-server/), an all-in-one WireGuard VPN solution with a web ui for connecting devices. Available at [services.wg-access-server](#opt-services.wg-access-server.enable).
 
+- [Actual Budget](https://actualbudget.org/), a local-first personal finance app. Available as [services.actual](#opt-services.actual.enable).
```

Comment:
Do you want the module to be backported? If not, it should go in the 25.05 rl notes.

Reply:
It's just been sitting for a while and 24.11 was released in the meantime. No need to backport.